db/migrate/20210610040352_create_oauth_access_grants.rb

```diff
@@ -0,0 +1,19 @@
+class CreateOauthAccessGrants < ActiveRecord::Migration[6.0]
+  def change
+    create_table :oauth_access_grants do |t|
+      t.references :resource_owner,  null: false
+      t.references :application,     null: false
+      t.text     :token,             null: false
+      t.integer  :expires_in,        null: false
+      t.text     :redirect_uri,      null: false
+      t.datetime :created_at,        null: false
+      t.datetime :revoked_at
+      t.string   :scopes,            null: false, default: ''
```

Comment:
Scope can be empty also in the Oauth2. So I think we don't need `null: false, default: ''`. What do you think?
```suggestion
      t.string   :scopes
```

Reply:
I just generated this from Doorkeeper
https://github.com/doorkeeper-gem/doorkeeper/blob/main/lib/generators/doorkeeper/templates/migration.rb.erb#L29

I think it's ok to keep it